### DISCUSSION TITLE
QSO Export

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#208] [Ideas]
One consideration of mine was to use the QSL Send Status for paper QSL's "Queued" as a marker for a later printout of the QSL cards.

The integration of a print program in QLog I do not consider useful.

But one could add a selection filter in the export dialog to export all records with "QSL Send Status Paper = Cueued". An additional date range is already available in the dialog. It might be necessary to specify a special reduced number of ADIF fields for this purpose.

The exported records could be used for your own printout of QSL cards or labels. Or you could use them for a QSL printing service as offered by DARC and others. https://qslshop.de/ADIF-processing-and-printing

There are also some external tools that allow QSL label printing. https://www.dxzone.com/catalog/Software/QSL/

Whether and how to implement this would have to be considered. I could imagine that the use of a QSL printing service could be interesting for some users.

The addition of the export dialog is certainly possible with a manageable effort. One would have to give the QLog user however some references, how he can handle the exported data sets further, in particular if he would like to print the QSL labels himself.

A CSV export would also facilitate further processing and label printing with LibreOffice Writer or Word.

**Comment by foldynl** [comment]
when we were talking about the Sent Status field , I made the same note as you write here. I plan to implement it in v0.24 (currently v0.23 is almost finished).
